releasenotes/notes/fix-scheduler-memory-slot-mapping-db299e85aed88037.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fixed basic scheduler such that the scheduler respects MemorySlots
+    in QuantumCircuit measures.
```

Comment:
Should also mention that measure stimulus pulses will only play on qubits which were explicitly measured, rather than all the qubits in the `meas_map` group.